CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## **[Unreleased]**
 
+- [#1401](https://github.com/wasmerio/wasmer/pull/1401) Make breaking change to `RuntimeError`: `RuntimeError` is now more explicit about its possible error values allowing for better insight into why a call may have failed.
```

Comment:
Suggest removing "may have". The usage of `may have` here could read as if the call may or may not have failed, I think you meant it as in the more explicit error may have more details, but that's already couched in the first half of the sentence.